docker-compose.yaml

```diff
@@ -0,0 +1,65 @@
+version: '2'
+services:
+  app:
+    build:
+      context: .
+    volumes:
+      - .:/app
+      - /app/node_modules
+    working_dir: /app
+    environment:
+      - SELENIUM_URL=http://selenium:4444/wd/hub
+      # To have report.ci defined
+      - TRAVIS=1
+      - TRAVIS_JOB_NUMBER=dummy
+      - TRAVIS_BRANCH=dummy
+      - TRAVIS_COMMIT=dummy
+      - TRAVIS_TAG=dummy
+      - TRAVIS_REPO_SLUG=dummy
+      - TRAVIS_COMMIT_MESSAGE=dummy
+      - TRAVIS_BUILD_ID=dummy
```

Comment:
Tests always fail on local environments without these definitions.
Definitions like `- TRAVIS` (without `=xxxx`) allows inherit specified environment variables and make tests pass only on TravisCI, or you can have them it pass locally if you define those variables. Please let me know if you like that.